docs/v13/documentation/install/create-a-prototype.md.njk

```diff
@@ -43,6 +43,11 @@ If any of your folder names contain spaces, you must add quotation marks around
 
 `cd ~/"a folder name with spaces/Documents/prototypes/juggling-licence"`
 
+### Navigate to your prototype folder using Windows
+
+Open your prototype folder in **File Explorer**, then right click and select **Open in terminal at this location**.
+
+
```

Comment:
We want to remove for now and write extra guidance for all platforms (maybe with their own pages) if it comes up more in research
```suggestion
```